packages/perseus-editor/src/components/locked-line-settings.tsx

```diff
@@ -26,6 +26,8 @@ import type {
     LockedPointType,
 } from "@khanacademy/perseus";
 
+const lengthZeroStr = "The line cannot have length 0.";
```

Comment:
This string doesn't need to be added to `PerseusStrings` for translation, does it? Do we have a policy of not doing i18n for the editor?

Reply:
I did a quick search, and it doesn't look like any of the editor strings are translated anywhere else.

Reply:
Confirmed. We don't translate the content editor.